edml-1.0.0.json

```diff
@@ -0,0 +1,215 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://schemas.exasol.com/edml-1.0.0.json",
+  "definitions": {
+    "varcharColumnSize": {
+      "type": "integer",
+      "default": 254,
+      "minimum": 1
```

Comment:
I am wondering whether we should add a maximum too.
Exasol has a limit (2M characters) and we could reflect that here.

Reply:
```suggestion
      "minimum": 1,
      "maximum": 2000000
```